src/librustc_middle/mir/interpret/error.rs

```diff
@@ -65,12 +64,12 @@ impl<'tcx> fmt::Display for FrameInfo<'tcx> {
             if tcx.def_key(self.instance.def_id()).disambiguated_data.data
                 == DefPathData::ClosureExpr
             {
-                write!(f, "inside call to closure")?;
```

Comment:
My original intention was to mean "inside *this* call to", but I guess that's obvious from the call site being displayed

Reply:
But that text was always attached to the call site, not the inside of the call.